provisioning/roles/chrome/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+- name: Download latest chrome
+  get_url: url={{ chrome_download_url }} dest=/tmp/chrome.deb
+  sudo: yes
+
+- name: Install Chrome
+  apt: deb=/tmp/chrome.deb state=present
+  sudo: yes
+
+- name: Download Chromedriver
+  get_url: url={{ chromedriver_download_url }} dest=/opt/chromedriver.zip
+  sudo: yes
+
+- name: Create directory for chromedriver
+  file: path=/opt/chromedriver state=directory
+  sudo: yes
+
+- name: Unzip Chromedriver
+  unarchive: src=/opt/chromedriver.zip dest=/opt/chromedriver/
+  sudo: yes
+
+- name: Set PATH variables
```

Comment:
Couldn't you set a symlink to `/opt/chromedriver/chromedriver` in `/usr/local/bin` instead?